src/Microsoft.Health.Dicom.Api/Registration/Swagger/SwaggerDefaultValues.cs

```diff
@@ -0,0 +1,45 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Linq;
+using EnsureThat;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Microsoft.Health.Dicom.Api.Registration.Swagger
+{
+    public class SwaggerDefaultValues : IOperationFilter
+    {
+        public void Apply(OpenApiOperation operation, OperationFilterContext context)
+        {
+            EnsureArg.IsNotNull(operation, nameof(operation));
+            EnsureArg.IsNotNull(context, nameof(context));
+
+            var apiDescription = context.ApiDescription;
+            operation.Deprecated |= apiDescription.IsDeprecated();
+
+            if (operation.Parameters == null)
+                return;
+
+            foreach (var parameter in operation.Parameters)
+            {
+                var description = apiDescription.ParameterDescriptions.First(p => p.Name == parameter.Name);
+                if (parameter.Description == null)
```

Comment:
Looks like this code is about backfilling from the data model section of the document if it's not populated in the method's metadata? Is this for the Swagger UI, the generated doc, or both?

Reply:
I believe it is for both, there were some open github issues and this was the suggested solution. I updated the file to match the solution provided in the repo: https://github.com/microsoft/aspnet-api-versioning/blob/master/samples/aspnetcore/SwaggerSample/SwaggerDefaultValues.cs